README.md

```diff
@@ -15,41 +15,60 @@ resources:
       endpoint: dirigeants # This is likely your github username or organization name
 ```
 
-**Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other repos.
-You can do this by going to your Project Settings (in azure devops):
+> **Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other
+repos.
+
+You can do this by going to your Project Settings (in Azure DevOps):
```

Comment:
these lines are still a part of the note, idk if you can do the ordered list inside the block quote though

Reply:
Fixed in 07c583d

Yeah, looks like we can block quote them as long as you put a `>` at the start of every line.